### PR TITLE
KOGITO-1799: kogito-runtimes: build using Java 8 compatibility

### PR DESCRIPTION
**Pull request by evacchi**
related apps https://github.com/kiegroup/kogito-apps/pull/169

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -956,7 +956,6 @@
           <configuration>
             <showDeprecation>true</showDeprecation>
             <showWarnings>true</showWarnings>
-            <release>${maven.compiler.release}</release>
```
Comment (radtriste) [inline pom.xml:959]: so this will make the compilation to be on Java 8 ?
where is set the fact that we want this to be compiled with Java 8 ?

Comment (evacchi) [reply pom.xml:959]: apparently i forgot a commit...

Comment (radtriste) [reply pom.xml:959]: why don't you use the same `maven.compiler.release` property ?

Comment (evacchi) [reply pom.xml:959]: only because it wouldn't build with JDK 8 otherwise. But i guess we can ignore that

**Pull request merged**